packages/server-nodejs/src/configuration/RuntimeConfiguration.ts

```diff
@@ -1,30 +1,40 @@
 
-import { IsOptional, IsUrl } from 'class-validator';
-
-import GatewayConfiguration from './GatewayConfiguration.js';
-import NodeConfiguration from './NodeConfiguration.js';
-import ProxyConfiguration from './ProxyConfiguration.js';
-import RepositoryConfiguration from './RepositoryConfiguration.js';
-import StandaloneConfiguration from './StandaloneConfiguration.js';
+import { z } from 'zod';
+
+import GatewayConfiguration, { schema as gatewaySchema } from './GatewayConfiguration.js';
```

Comment:
Should we rename the exports of the schemas to avoid renaming them in the import?

Reply:
Processed